wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Content/Text/ConversationQuoteCell.swift

```diff
@@ -37,12 +37,12 @@ final class ConversationReplyContentView: UIView {
 
         var showDetails: Bool {
             guard let message = quotedMessage,
-                  (message.isText
+                  message.isText
```

Comment:
Looking at all the checks, I think `isFile` check is good enough for this logic

Reply:
let's explore that on a separate PR. On this one, I would like to focus on fixing the warnings.